### DISCUSSION TITLE
Difference between merged and not merged process atomic dataset?

### DISCUSSION THREAD

**Discussion by Jeriousman** [RUCAIBox/RecBole#1596] [General] [answered]
When I download for example processed Diginetica dataset from google drive of urs, I get merged and not_merged files. What are the differences?

**Comment by Sherry-XLL** [comment, marked as answer]
@Jeriousman Hello, interactions between the same pair of user and item may appear more than once, e.g. one user purchases the same kind of bread many times.

Thus, merged means we remain at most one interaction for a pair of user and item (also means have removed duplicated user-item interactions) after generating the atomic files, and not_merged vice versa.

For more details, you may also refer to https://github.com/RUCAIBox/RecBole/issues/487.

**Reply by Jeriousman** [reply]
Thank you for your prompt answer. Maybe it is really nice for Recbole that you write this info in README of the atomic file. This information is missing.
